opm/input/eclipse/Generator/KeywordGenerator.cpp

```diff
@@ -17,83 +17,98 @@
   along with OPM.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#include <fstream>
-#include <filesystem>
-#include <iostream>
-#include <sstream>
-#include <cctype>
-#include <fmt/format.h>
-
-#include <opm/json/JsonObject.hpp>
 #include <opm/input/eclipse/Generator/KeywordGenerator.hpp>
+
 #include <opm/input/eclipse/Generator/KeywordLoader.hpp>
+
+#include <opm/json/JsonObject.hpp>
+
 #include <opm/input/eclipse/Parser/ParserKeyword.hpp>
 
+#include <cctype>
+#include <filesystem>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <map>
+
+#include <fmt/format.h>
 
 namespace {
 
-const std::string sourceHeader = R"(
+    void updateFile(const std::string& newContent, const std::string& filename)
+    {
+        Opm::KeywordGenerator::ensurePath(filename);
+
+        std::ofstream { filename } << newContent;
+    }
+
+    void write_file(const std::string& content,
+                    const std::string& file,
+                    const bool         verbose,
+                    const std::string& desc = "source")
```

Comment:
i think we can drop the default?

Reply:
> i think we can drop the default?

True.  I've pushed an update to that effect.